src/requests/communications.js

```diff
@@ -0,0 +1,119 @@
+// File: communications.js
+
+const axios = require('axios')
+const apiEndpoints = require('@constants/endpoints')
+
+const baseUrl = process.env.COMMUNICATION_BASE_URL || 'http://communications:3123'
```

Comment:
@nevil-mathew don't make this is optional, if CHAT is enabled this has to set in the env.